packages/abort-controller/LICENSE

```diff
@@ -186,7 +186,7 @@ Apache License
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Referencing from LICENSE in aws-cdk ([link](https://github.com/aws/aws-cdk/blob/fb3fc4813c644e724fd1dea0d4178fd7bd543f24/packages/%40aws-cdk/aws-dynamodb-global/lambda-packages/aws-global-table-coordinator/LICENSE#L189))
```suggestion
   Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Reply:
Fixed in 02b120b